diff --git a/node_modules/tempusdominus-bootstrap-4/build/js/tempusdominus-bootstrap-4.js b/node_modules/tempusdominus-bootstrap-4/build/js/tempusdominus-bootstrap-4.js
index 64d1f42..351ac52 100644
--- a/node_modules/tempusdominus-bootstrap-4/build/js/tempusdominus-bootstrap-4.js
+++ b/node_modules/tempusdominus-bootstrap-4/build/js/tempusdominus-bootstrap-4.js
@@ -410,6 +410,8 @@ var DateTimePicker = function ($, moment) {
                 if (!this._options.allowMultidate || this._dates.length === 1) {
                     this.unset = true;
                     this._dates = [];
+                    this._dates[0] = this.getMoment();
+                    this._viewDate = this.getMoment().locale(this._options.locale).clone();
                     this._datesFormatted = [];
                 } else {
                     outpValue = this._element.data('date') + ',';
